taskcluster/xpi_taskgraph/transforms/release_notifications.py

```diff
@@ -22,6 +22,8 @@ def add_notifications(config, jobs):
     xpi_name = config.params.get("xpi_name")
     xpi_revision = config.params.get("xpi_revision")
     shipping_phase = config.params.get("shipping_phase")
+    additional_shipit_emails = config.params.get("additional_shipit_emails")
```

Comment:
We may want to make this `.get("additional_shipit_emails", [])` just to make sure it's a list below, since the parameter is optional.